pages/PIV_Testing_Checklist.md

```diff
@@ -3,4 +3,281 @@ layout: default
 title: Common Issues Checklist for PIV/PIV-I Card Testing
 permalink: /PIV_Testing_Checklist/
 ---
-Placeholder for Common Issues Checklist to be uploaded.
+
+![GSA FIPS 201 logo]({{site.baseurl}}/img/FIPS201-logo.JPG){:style="float:left"}
+
+## Overview
+This guide provides a list of common issues found during recent PIV/PIV-I card testing.  These issues are listed for one of two reasons:
+
+*	The issue creates a FIPS 201-2 compliance issue and is a cause for failure of the annual FIPS 201 EP Testing
+*	The issue creates problems with PACS interoperability
+
+Before scheduling a date for in-person testing with the FIPS 201 EP program, please review the issues listed below and resolve any found to be on the sample production card to be used during testing.  A list of technical and policy guidance publications are included in the reference section of this document.
```

Comment:
we state FIPS 201 EP and FICAM testing interchangeably...be generic or one or the other (ficam testing)?